jwcrypto/jwk.py

```diff
@@ -353,6 +366,49 @@ def _import_pyca_pub_ec(self, key, **params):
         )
         self.import_key(**params)
 
+    def _generate_OKP(self, params):
+        curve = 'Ed25519'  # Ed25519 is the default Edwards curve
+        if 'curve' in params:
+            curve = params.pop('curve')
+        # 'curve' is for backwards compat, if 'crv' is defined it takes
```

Comment:
given this is a new type please let's drop the old "curve" parameter support and use exclusively "crv"

Reply:
Agreed, will do.